data/mods/Spookdex/abilities.js

```diff
@@ -4308,6 +4308,53 @@ let BattleAbilities = {
 		rating: 2.5,
 		num: 139,
 	},
+	"ironleg": {
+		desc: "This Pokemon's punch-based attacks have their power multiplied by 1.2.",
+		shortDesc: "This Pokemon's kick-based attacks have 1.2x power.",
+		onBasePowerPriority: 8,
+		onBasePower: function (basePower, attacker, defender, move) {
+			if (move.flags['kick']) {
```

Comment:
'kick' is not a flag, so this won't work

Reply:
i gave all the kicking moves a kick flag, so it should check for it. Unless im wrong.